AzureIotPnpDps.X/platform/application_manager.c

```diff
@@ -176,37 +183,23 @@ uint32_t MAIN_dataTask(void* payload)
 	{
 		// How many seconds since the last time this loop ran?
 		int32_t delta = difftime(timeNow, previousTransmissionTime);
-		if (delta >= CFG_SEND_INTERVAL)
+		if (delta >= telemetry_interval_seconds)
 		{
 			previousTransmissionTime = timeNow;
 
 			// Call the data task in main.c
 			fpSendToCloudCallback();
 		}
+		check_led_status(NULL);
+		check_button_status();
 	}
 
-	if (shared_networking_params.haveAPConnection)
-	{
-		LED_BLUE_SetLow();
-	}
-	else
-	{
-		LED_BLUE_SetHigh();
-	}
-
-	if (LED_isBlinkingGreen() == false)
-	{
-		if (CLOUD_isConnected())
-		{
-			LED_GREEN_SetLow();
-		}
-		else
-		{
-			LED_GREEN_SetHigh();
-		}
-	}
+//	if (shared_networking_params.haveAPConnection)
+//	{
+		// Add code for periodic operation when AP mode is enabled and have connection 
+//	}
```

Comment:
just checking that this is also supposed to be commented

Reply:
This if section was there as a placeholder but there is no code to execute.  Keeping them with comment out as a placeholder.